build.sbt

```diff
@@ -59,6 +63,14 @@ lazy val chronoscala = (project in file("."))
           }
         }
       },
+      //the exclusion filter is necessary for Scala compiler bug
```

Comment:
Please also include the link to this issue: https://github.com/scala/bug/issues/11207, and mention about 0.3.0 is compiled with 2.12.7. like:
```
//chronoscala 0.3.0 is compiled with Scala 2.12.7.
//the exclusion filter is necessary to avoid false positive of binary compatibility issue for Scala compiler bug (https://github.com/scala/bug/issues/11207).
```